tests/test_tdag.py

```diff
@@ -1,42 +1,6 @@
 import pytest
-from polytracker import dumptdag, taint_dag, ProgramTrace
-from os import getcwd
+from polytracker import taint_dag, ProgramTrace
 from typing import cast
-from .data import TEST_DATA_DIR, TEST_DATA_PATH, TEST_RESULTS_DIR
-from polytracker.containerization import CAN_RUN_NATIVELY
-
-
-@pytest.mark.program_trace("test_tdag.cpp")
-def test_dumptdag(program_trace: ProgramTrace):
```

Comment:
Do we have any other test that checks these details? If not, I think we should rewrite this test case to use the new api.

Reply:
There's test for the API version in the same file IIRC.